packages/cli/src/scaling/pubsub/pubsub-handler.ts

```diff
@@ -20,10 +24,13 @@ export class PubSubHandler {
 		private readonly eventbus: MessageEventBus,
 		private readonly externalSecretsManager: ExternalSecretsManager,
 		private readonly communityPackagesService: CommunityPackagesService,
+		private readonly publisher: Publisher,
+		private readonly workerStatus: WorkerStatus,
 	) {}
 
 	init() {
 		if (this.instanceSettings.instanceType === 'webhook') this.setupWebhookHandlers();
+		if (this.instanceSettings.instanceType === 'worker') this.setupWorkerHandlers();
```

Comment:
Could we maybe make this an exhaustive switch case? I don't think we'll be adding any more work types soon, but I wouldn't mind this signalling an error if we do.